torch/csrc/jit/interpreter.cpp

```diff
@@ -625,6 +627,10 @@ struct CodeImpl {
     size_t r = type_table_.size();
     type_table_.emplace_back(std::move(t));
     return r;
+  void emitProfile(Node* node) {
+    emitLoadInputs(node->inputs());
+    insertInstruction(PROFILE, profile_function_table_.size());
+    profile_function_table_.push_back(node->cast<ProfileOp>()->getCallback());
```

Comment:
Do we need a separate profile_function_table_? Can it use the op table like OP and OPN?